docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -90,7 +90,7 @@ Constructors
     Array elements must not be ``NULL`` or empty.
     The returned geometry may not be simple and may contain duplicate points if input array has duplicates.
 
-.. function:: ST_Point(double, double) -> Point
+.. function:: ST_Point(longitude, latitude) -> Point
```

Comment:
This has to stay as the data type. Maybe we have to clarify in the description.. see how this is done in other functions in this document or elsewhere in the trino docs

Reply:
ST_point and other geometry related functions don’t operate on longitude and latitude. They work on points on an Euclidean plane. 

You need to use the SphericalGeography related functions to use longitude and latitude.